packages/bpk-component-barchart/src/customPropTypes-test.js

```diff
@@ -35,7 +37,7 @@ const badData = [
 describe('chartDataProp', () => {
   it('should fail if data is not an array', () => {
     const result = chartDataProp(
-      { data: {}, yScaleDataKey: 'price' },
+      { data: {}, xScaleDataKey: 'day', yScaleDataKey: 'price' },
```

Comment:
I see a new argument's been added here — why is that?

Reply:
Yes this was due to the test being setup incorrectly if you look at the description and the component actually requires this prop so correcting the implementation